packages/suite/src/components/suite/layouts/SuiteLayout/PageHeader/PageNames/AccountName/AccountDetails.tsx

```diff
@@ -110,6 +111,7 @@ export const AccountDetails = ({ selectedAccount, isBalanceShown }: AccountDetai
                     payload={{
                         type: 'accountLabel',
                         entityKey: key,
+                        accountType,
```

Comment:
Please pass it as normal prop, do not include it into `payload`. By that you do not need to edit `metadataTypes` and avoid passing it as string 🙏